Description: Only run these tests if REAL_SERVERS is set.
 Tests need internet accesss. Solution copied from t/real-servers.t.
Origin: vendor
Bug-Debian: http://bugs.debian.org/728588
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2013-11-03

--- a/t/proxy-with-https.t
+++ b/t/proxy-with-https.t
@@ -3,6 +3,10 @@
 use URI::Escape;
 
 use Test::More;
+
+plan skip_all => "enable these tests by setting REAL_SERVERS"
+  unless $ENV{REAL_SERVERS};
+
 use HTTP::Request;
 
 require 't/TestServer.pm';
